lib/kamal/configuration/docs/builder.yml

```diff
@@ -37,6 +37,16 @@ builder:
     arch: arm64
     host: ssh://docker@docker-builder
 
+  # Buildpack configuration
+  #
+  # The build configuration for using pack to build a Cloud Native Buildpack image.
```

Comment:
Add mention of project.toml to set your excluded options. https://buildpacks.io/docs/for-app-developers/how-to/build-inputs/use-project-toml/

Reply:
As I was thinking about this and removing `context: "."` it doesn't matter as much since it's using the git clone. The exclusion list is really only relevant when you're using the local context.